bin/env_variables_all.txt

```diff
@@ -1,5 +1,12 @@
 APPINSIGHTS_INSTRUMENTATIONKEY=SomeRandomStringForLocalDocker
 BEFTA_RESPONSE_HEADER_CHECK_POLICY=JUST_WARN
+BEFTA_OAUTH2_ACCESS_TOKEN_TYPE_OF_XUIWEBAPP=OIDC
+BEFTA_OAUTH2_CLIENT_ID_OF_XUIWEBAPP=xuiwebapp
+BEFTA_OAUTH2_REDIRECT_URI_OF_XUIWEBAPP=http://localhost:3333/oauth2/callback
+BEFTA_OAUTH2_CLIENT_SECRET_OF_XUIWEBAPP=OOOOOOOOOOOOOOOO
```

Comment:
Can we retain the alphabetical order of the file, i.e. move this line up 1, thanks.